administration/src/bp-modules/cards/ImportCardsRequirementsText.tsx

```diff
@@ -18,15 +19,31 @@ const ImportCardsRequirementsText = ({
   csvHeaders,
   isFreinetFormat = false,
 }: ImportCardsRequirementsProps): ReactElement => {
+  const projectConfig = useContext(ProjectConfigContext)
+  const { card: cardConfig } = projectConfig
   const { t } = useTranslation('cards')
+
+  const requiredHeaders = useMemo(() => {
+    const headers = [cardConfig.nameColumnName, cardConfig.expiryColumnName]
+    cardConfig.extensions.forEach((extension, index) => {
+      const header = cardConfig.extensionColumnNames[index]
+      if (extension.isMandatory && header) {
```

Comment:
🔧 i would prefer explicit null check here
```suggestion
      if (extension.isMandatory && header !== null) {
```